### PR TITLE
Add !restart command

### PR DESCRIPTION
**Pull request by buffet**
1. Relying on systemd for restarting is sufficient right?
2. Is os.Exit() the correct way to exit with async stuff going on?

### CODE REVIEW

**Comment by elkowar** [comment]
Lgtm, but idk if any more cleanup would be necessary, we should wait for feedback from @mlvzk on that

**Review by mlvzk — APPROVED** [review]

**Pull request merged**
